index.py

```diff
@@ -20,13 +21,24 @@ def __init__(self, cmd_args):
         mqtt.connect(not cmd_args.offline)
         self.modules = None
         self.states = {}
+        self.socket = Socket(self)
+        self.socket.start()
 
     def import_module(self, loc: str) -> (Optional[Any], str):
         name = path.basename(loc).replace('.mod.py', '', -1)
         spec = importlib.util.spec_from_file_location(name, loc)
         module = spec.loader.load_module()
         return module, name
 
+    def get_game(self, game):
+        self.read_states()
+        if game not in self.states:
+            return None
+        state = self.states[game]
+        if not state.is_game:
```

Comment:
```suggestion
        if not state.player_class:
```
en dan player_class = None toevoegen aan BaseState.__init__() (wat sws wel handig is)